doc/lsst.source.injection/reference/11_make_injection_pipeline.rst

```diff
@@ -300,11 +300,11 @@ The output PNG file from the above example is shown below:
 .. list-table::
     :widths: 1 1 1
 
-    * - .. image:: ../_assets/DRP-RC2+injection_step1_inject_exposure.png
+    * - .. image:: ../_assets/DRP-RC2\+injection_step1_inject_exposure.png
             :width: 100%
-      - .. image:: ../_assets/DRP-RC2+injection_step1_inject_visit.png
+      - .. image:: ../_assets/DRP-RC2\+injection_step1_inject_visit.png
```

Comment:
If that doesn't work try `%2B` instead of `+`?

Reply:
Thanks. It looks like that may be the problem - more discussion in this Slack thread: https://lsstc.slack.com/archives/C2B6DQBAL/p1699542611708879?thread_ts=1699542611.708879&cid=C2B6DQBAL